src/qt/rpcconsole.cpp

```diff
@@ -486,8 +486,18 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
         ui->openDebugLogfileButton->setIcon(platformStyle->SingleColorIcon(":/icons/export"));
     }
     ui->clearButton->setIcon(platformStyle->SingleColorIcon(":/icons/remove"));
+
     ui->fontBiggerButton->setIcon(platformStyle->SingleColorIcon(":/icons/fontbigger"));
+    //: Main shortcut to increase the RPC console font size.
+    ui->fontBiggerButton->setShortcut(tr("Ctrl++"));
+    //: Secondary shortcut to increase the RPC console font size.
```

Comment:
Maybe add a note that a secondary shortcut differs from a main one only by presence/absence of the \<Shift> key in the key sequence?

Reply:
I think this is ok as is. If we include that i'd be worried if someone translates into `Ctrl+Shift+_`.